packages/astro/src/transitions/router.ts

```diff
@@ -373,6 +384,9 @@ async function transition(
 }
 
 export function navigate(href: string, options?: Options) {
+	
+	if (!supportsViewTransitions) return;
+	
```

Comment:
Should we throw an error on SSR instead? Or maybe remove this completely? I think there's a bigger problem if the user is calling these client APIs in SSR.

If view transitions is not supported in the client, it's already handled below at `if (!transitionEnabledOnThisPage()) {`

Reply:
The check mostly serves as an opportunity for tree-shaking.

I went back and forth on no-op, error and warning. Ultimately, it's a very unlikely scenario that `navigate()` gets called without user interaction so I stopped thinking about it.

I already wrote the docs saying it does nothing on the server. I'd rather not go back to that PR. Would a warning be acceptable here?

Reply:
Yeah a warning would be fine by me if it's already documented as a no-op 👍